haskell/binutils-repo.bzl

```diff
@@ -0,0 +1,39 @@
+"""Rule for a repo containing an auxiliary toolchain with binutils."""
+
+def _locate_tool(ctx, tool_name):
+  full_path = ctx.which(tool_name)
+  if full_path == None:
+    fail("Could not locale {0}, make sure it's on PATH".format(tool_name))
```

Comment:
do you mean `locate` here?

Reply:
Yes, this certainly has nothing to do with locale! :-)